mono/mini/unwind.c

```diff
@@ -551,6 +551,8 @@ mono_unwind_frame (guint8 *unwind_info, guint32 unwind_info_len,
 				   host_mgreg_t **save_locations, int save_locations_len,
 				   guint8 **out_cfa)
 {
+	MONO_SCOPE_ENABLE_JIT_WRITE();
+
```

Comment:
This doesn't even generate code.

Reply:
yea i'm surprised this is here.  No way to know without a dev kit.